webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Button, noop } from 'patternfly-react';
+
+const InsightsCloudSync = ({ syncInsights }) => (
+  <IntlProvider locale={navigator.language}>
```

Comment:
can you set here the document title
`document.title = INSIGHTS_SYNC_PAGE_TITLE;`